projects/aca-shared/rules/src/app.rules.ts

```diff
@@ -624,3 +626,19 @@ export function canOpenWithOffice(context: AcaRuleContext): boolean {
 
   return context.permissions.check(file, ['update']);
 }
+
+function isSmartFolder(context: RuleContext): boolean {
+  if (context.selection && !context.selection.isEmpty) {
+    const node = context.selection.first;
+    if (!node?.entry.isFolder) {
+      return false;
+    }
+    const nodeAspects = getNodeAspectNames(node);
+    return nodeAspects.includes('smf:customConfigSmartFolder') || nodeAspects.includes('smf:systemConfigSmartFolder');
+  }
+  return false;
+}
+
+function getNodeAspectNames(node: NodeEntry): string[] {
```

Comment:
Do we need a separate function for that? It's just one line with no side effects and we're not reusing that anywhere else for now